### ISSUE TITLE
stats.ctime cannot be used to retrieve the creation time

### ISSUE THREAD

**Issue by KasparEtter** [jhotmann/node-rename-cli#10]
First of all: This is a really awesome tool and _exactly_ what I was looking for! 😍

https://github.com/jhotmann/node-rename-cli/blob/5d75fdcfc774796efd6a5f830cad9815f6323f20/lib/replacements.js#L264-L271

[ctime](https://linuxhandbook.com/file-timestamps/):
> `ctime` stands for status change time. This timestamp tells you when was the last time the property and metadata of the file were changed. The metadata includes file permissions, ownership, name and location of the file.

Please use [`stats.birthtime`](https://nodejs.org/api/fs.html#fs_class_fs_stats) for this (see [this documentation](https://nodejs.org/api/fs.html#fs_stat_time_values) for platform availability).

In other words, with `{{date|create}}` you get when the file content _or its name_ was last changed. Until this is fixed, I will just use `{{date|modify}}` instead, which at least ignores file renames.

**Comment by jhotmann** [comment]
Glad you like the tool! I am going to make this change. If you run into things like this continue to submit issues but in the meantime you can also override the default behavior with the replacements.js file that is installed to `~/.rename/replacements.js` or `%UserProfile%\.rename\.replacements.js` on Windows. Just copy the entire `date` object and replace `ctime` with `birthtime`.

**Comment by jhotmann** [comment]
Fixed in 6.1.0

**Comment by KasparEtter** [comment]
Awesome, thanks a lot for addressing all these issues/suggestions! ❤️
